src/sidebar/components/thread-list.js

```diff
@@ -13,14 +13,25 @@ import {
 
 import ThreadCard from './thread-card';
 
+/** @typedef {import('../util/build-thread').Thread} Thread */
+
 // The precision of the `scrollPosition` value in pixels; values will be rounded
 // down to the nearest multiple of this scale value
 const SCROLL_PRECISION = 50;
 
 function getScrollContainer() {
-  return document.querySelector('.js-thread-list-scroll-root');
+  const container = document.querySelector('.js-thread-list-scroll-root');
+  if (!container) {
+    throw new Error('Scroll container is missing');
+  }
+  return container;
 }
 
+/**
+ * @typedef ThreadListProps
+ * @prop {Thread} thread - Should annotations render extra document metadata?
```

Comment:
```suggestion
 * @prop {Thread} thread
```

The prop documentation refers to something else.